src/index.ts

```diff
@@ -104,10 +104,23 @@ function fileToText(file: File): Promise<string> {
 }
 
 class MultinetAPI {
-  private client: Client;
+  public client: Client;
```

Comment:
Why was this change necessary?

Reply:
The methods `setAuthToken` and `removeAuthToken` are added onto the client, as it's not really an API call and didn't seem correct to add these to the `MultinetAPI` class. The `setAuthToken` function is used [here](https://github.com/multinet-app/multinet-client/blob/3e09b1d19cbb0e9627e3739fce90b9f41a112d94/src/components/LoginMenu.vue#L142), which requires that client is public.

Reply:
The problem is, the token *is* part of the public interface already: you've added a token argument to the constructor. Since that's the case, I don't have a problem with also adding set/remove methods for that token to the interface as well.

I'm not opposed to making `client` a public member of `MultinetAPI`, but I think that's a major change to the intended API and so ought to be considered independently. In this case, it seems odd that everything else is done via the `MultinetAPI` interface, but setting the token value used for all authentication in the other methods requires reaching into the implementation layer.

For now I'd suggest just adding the `setAuthToken()` and `removeAuthToken()` methods to the public interface of `MultinetAPI`. It's ok that they are not calls into the Multinet REST API. This class is meant to be a an easy way to call into the REST API, but that doesn't mean it can ONLY make those calls.

Reply:
I've addressed your concerns, and rebased to revert the commit that made the api client member public.